server/lib/nl/detection/date.py

```diff
@@ -25,6 +25,8 @@
     r'(in|after|on|before|year)(?: year)? (\d{4})',
 ]
 
+YEARS_AGO_RE = [r'(decade) ago', r'(\d+) years ago']
```

Comment:
Not for implementing in this PR: Do we plan to add support for queries using words instead of digits ("what was the population of California ten years ago")?

Reply:
I think we will want to keep improving these & hopefully move away from regex and towards using a model to tell us what dates are in the query